dotnet/src/SemanticKernel.Core/Extensions/FunctionViewExtensions.cs

```diff
@@ -9,7 +9,7 @@ namespace Microsoft.SemanticKernel.Extensions;
 /// <summary>
 /// Extensions for functions views.
 /// </summary>
-public static class FunctionViewExtensions
+public static class SKFunctionMetadataExtensions
```

Comment:
nit: the file name still has old name - 'FunctionViewExtensions.cs'

Reply:
Oops, I didn't see this. Will rename it.

Reply:
Actually, let me do this in a follow-up PR, since it's minor and I have two signoffs :)

Reply:
Nevermind, I needed to fix a merge conflict anyway; fixed.